docs/introduction/get-started/p2p-private-offchain-txs.md

```diff
@@ -0,0 +1,97 @@
+In this section, we show you how to make off-chain transactions and send funds to another account using the Miden client. 
+
+!!! important "Prerequisite steps"
+    - You should have already followed all previous sections.
+    - You should have *not* reset the state of your local client. 
+
+## Create a second account
+
+!!! tip
+      Remember to use the [Miden client documentation](https://docs.polygon.technology/miden/miden-client/cli-reference/) for clarifications.
+
+1. Create a second account to send funds with. Previously, we created a `basic-immutable` (account A). Now, create `basic-immutable` (account B) using the following command:
+
+      ```shell
+      miden-client account new basic-immutable
+      ```
+
+2. List and view the newly created accounts with the following command:
+
+      ```shell
+      miden-client account -l
+      ```
+
+3. You should see two accounts:
+
+      ![Result of listing miden accounts](../../img/get-started/two-accounts.png)
+
+## Transfer assets between accounts
+
+1. Now we can transfer some of the tokens we received from the faucet to our second account B. 
+
+    To do this, run:
+
+    ```shell
+    miden-client tx new p2id <regular-account-id-A> <regular-account-id-B> <faucet-account-id> 50
+    ```
+
+    !!! note
+        The faucet account id can be found on the [Miden faucet website](https://ethdenver.polygonmiden.io/) under the title **Miden faucet**.
+
+    This generates a Pay-to-ID (`P2ID`) note containing `<amount>` assets, transferred from one account to the other. 
+
+2. First, sync the accounts.
+
+    ```shell
+    miden-client sync # Make sure we have an updated view of the state
+    ```
+
+3. Get the second note id.
+
+    ```sh
+    miden-client input-notes list 
+    ```
+
+4. Have the second account consume the note.
+
+    ```sh
+    miden-client tx new consume-notes <regular-account-ID-B> <input-note-id> 
```

Comment:
also here, we can use a short version of the note id now. 

@igamigo knows how short (I think its 7 chars after 0x)

Reply:
Please confirm and I will add.

Reply:
As long as the input prefix matches only one note it will work. With 7 characters it will very likely always match only one note.